packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -105,6 +115,11 @@ export default class FetchCache implements CacheHandler {
         }
       )
 
+      if (res.status === 429) {
+        const retryAfter = res.headers.get('retry-after') || '60'
+        rateLimitedUntil = Date.now() + parseInt(retryAfter)
```

Comment:
Is retry after in seconds or ms already?

Reply:
updated to return ms